profilegen/src/res/templates/client_template.template

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+<%- @file_names.each do |file_name| -%>
+require_relative 'modules/<%= file_name %>'
+<%- end -%>
+require_relative 'utils/configurable'
+require_relative 'utils/default'
+
+module Azure
+  module Management
+    module Profiles
+      module <%= @profile_name %>
+        #
+        # Client class for the <%= @profile_name %> profile SDK.
+        #
+        class Client
+          include Azure::ARM::Configurable
+
+          attr_accessor <% @module_class_mapper.each_with_index do |module_class, index| %> :<%= module_class[:class_name].downcase %><% if index != @module_class_mapper.length - 1 %>,<% end %><% end %>
+
+          def initialize(options = {})
+            if options.is_a?(Hash) && options.length == 0
+              @options = setup_options
+            else
+              @options = options
+            end
+
+            reset!(options)
+
+            <%- @module_class_mapper.each do |module_class| -%>
+            self.<%= module_class[:class_name].downcase %> = <%= "#{module_class[:module_name]}::#{module_class[:class_name]}" %>.new(self, nil, nil)
+            <%- end -%>
+          end
+
+          def credentials
+            if @credentials.nil?
+              self.active_directory_settings ||= Azure::ARM::Default.active_directory_settings
+
+              @credentials = MsRest::TokenCredentials.new(
+                    MsRestAzure::ApplicationTokenProvider.new(
+                        self.tenant_id, self.client_id, self.client_secret, self.active_directory_settings))
+            end
+            @credentials
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
please leave an new line character at the end so that all generated file would have that as well

Reply:
Done